include/gc/gc_mark.h

```diff
@@ -362,6 +362,11 @@ GC_API int GC_CALL GC_is_marked(const void *) GC_ATTR_NONNULL(1);
 GC_API void GC_CALL GC_clear_mark_bit(const void *) GC_ATTR_NONNULL(1);
 GC_API void GC_CALL GC_set_mark_bit(const void *) GC_ATTR_NONNULL(1);
 
+/* Finalizer queue bit manipulation.                                    */
+GC_API int GC_CALL GC_is_finalizer_bit_set(const void *) GC_ATTR_NONNULL(1);
```

Comment:
I might have misunderstood the purpose but might this better be something like "GC_is_finalizer_queued_bit_set` or similar?

Reply:
Agreed. I shortened it because it was getting quite verbose. Fixed here: bb1cc334db4dedbf207c111717946dfd448d65f5